src/utility_windows.go

```diff
@@ -242,3 +242,18 @@ func getLocalUser(userName string) (shared.LocalUser, error) {
 	}
 	return shared.LocalUser{}, nil
 }
+
+func getLocalServiceStatus(serviceName string) (shared.Service, error) {
+    serviceDataList, err := wapi.GetServices()
+    var serviceStatusData shared.Service
+	if err != nil {
+        failPrint("Couldn't get local service: " + err.Error())
+        return serviceStatusData, err
+    }
+    for _, v := range serviceDataList {
+        if v.SCName == serviceName {
+            serviceStatusData = v
```

Comment:
Could probably just return the status at this point and save yourself some loops

Reply:
safinlint: what shane said: try:
```go
return serviceStatusData, nil
}
}
failPrint("Invalid service name: " + serviceName)
return nil, errors.New("Invalid service name: " + serviceName)
```